internal/provider/custom_policy.go

```diff
@@ -175,18 +175,12 @@ func (r *customPolicyResource) Create(ctx context.Context, req resource.CreateRe
 		return
 	}
 
-	var scopeMrn string
+	scopeMrn := "//platform.api.mondoo.app"
```

Comment:
The scope MRN should be
```suggestion
	scopeMrn := "//policy.api.mondoo.app"
```